source/agora/crypto/Schnorr.d

```diff
@@ -247,26 +310,26 @@ nothrow @nogc @safe unittest
 {
     Pair kp = Pair.fromScalar(Scalar(`0x074360d5eab8e888df07d862c4fc845ebd10b6a6c530919d66221219bba50216`));
     static immutable string message = "Bosagora:-)";
-    auto signature = sign(kp, message);
-    assert(verify(kp.V, signature, message));
+    auto sig = sign(kp, message);
+    assert(verify(kp.V, sig, message));
```

Comment:
Was this necessary ? Same for the next 2 unittests.

Reply:
Updated